src/scanner/scanner.py

```diff
@@ -297,41 +310,38 @@ async def _scan(self, geocode):
                 )
 
     async def _scan_all_geocodes(self):
-        tasks = [
-            self._scan(geocode)
-            for geocode in self.data.municipio_geocodigo.unique()
-        ]
+        tasks = []
```

Comment:
This can also be outsourced to duckdb (it can do this much faster than python:
```sql
select distinct municipio_geocodigo from <table>
```
where table can be a DF,  Parquet file, or anything else.

Reply:
This is not the problem, the query result that  composes self.data only contains geocodes for that specific UF
![image](https://github.com/AlertaDengue/episcanner-downloader/assets/82233055/0efd622d-0c4b-4c0f-9285-895c36631a61)